data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -65,7 +65,7 @@
       { "item": "glock_22", "prob": 35, "charges-min": 0 },
       { "item": "glock_29", "prob": 2, "charges-min": 0 },
       { "item": "glock_31", "prob": 15, "charges-min": 0 },
-      { "item": "glock_40", "variant": "glock_40", "prob": 2, "charges-min": 0 },
+      { "item": "glock_20", "variant": "glock_40", "prob": 2, "charges-min": 0 },
```

Comment:
I think you need to rebase...

Reply:
Did a goof/bad practice by copypasting code from an old save, fixed now.